docs/Configuration/Configuration.md

```diff
@@ -0,0 +1,149 @@
+﻿# KMD Logic Document Generation Configuration
+
+To configure a Logic Document Generation environment, you need to access [Logic Console](https://console.kmdlogic.io/) to do so.
```

Comment:
"to do so" can be removed.

Reply:
Fixed in deeb2f7950fd5fcc6858b8763b0c16761a2c9d91